### PR TITLE
calceph 4.0.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by gastineau** [comment]
The library switches to cmake.
So add
```
  
depends_on "cmake" => :build

```

replace the install step by 
```

def install
    system "cmake", "-S", ".", "-B", "build",  "-DBUILD_SHARED_LIBS=ON", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
  end

```

**Comment by cho-m** [comment]
```
Files with missing rpath:
  /opt/homebrew/Cellar/calceph/4.0.0/bin/calceph_inspector
  /opt/homebrew/Cellar/calceph/4.0.0/bin/calceph_queryorientation
  /opt/homebrew/Cellar/calceph/4.0.0/bin/calceph_queryposition
```

**Review by p-linnane — APPROVED** [review]

File: Formula/c/calceph.rb [hunk 1]
```diff
@@ -11,30 +11,27 @@ class Calceph < Formula
...
-                          "--disable-dependency-tracking",
-                          "--disable-silent-rules",
-                          "--prefix=#{prefix}"
-    system "make"
-    system "make", "install"
+    args = %w[
+      -DBUILD_SHARED_LIBS=ON
+      -DENABLE_FORTRAN=ON
+      -DCMAKE_INSTALL_RPATH=#{rpath}
+    ]
```
Comment (ZhongRuoyu) [inline Formula/c/calceph.rb:31]: This should be a `%W` array instead of `%w`. #169953

Comment (chenrui333) [reply Formula/c/calceph.rb:31]: make sense 👍

**Pull request merged**